src/modules/75-cf/pages/feature-flags/components/StaleFlagActions/__tests__/StaleFlagActionDialog.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { render, RenderResult, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { TestWrapper } from '@common/utils/testUtils'
+import mockFeatureFlags from '@cf/pages/feature-flags/__tests__/mockFeatureFlags'
+import * as cfServiceMock from 'services/cf'
+import { StaleFlagActionDialog, StaleFlagActionDialogProps } from '../StaleFlagActionDialog'
+
+const renderComponent = (props: Partial<StaleFlagActionDialogProps> = {}): RenderResult =>
+  render(
+    <TestWrapper>
+      <StaleFlagActionDialog markAsNotStale selectedFlags={[]} onAction={jest.fn()} {...props} onClose={jest.fn()} />
```

Comment:
You'll probably notice that the coverage won't be complete on this component and it'll highlight that the `onClose` stuff isn't being exercised. With the `{...props}` coming _before_ `onClose={jest.fn()}` you won't be able to pass a mock to the component to assert against since it'll always get overridden.

It's passing the coverage checks, so don't worry about it in this PR, but something to think about when we're next in this area.

```suggestion
      <StaleFlagActionDialog markAsNotStale selectedFlags={[]} onAction={jest.fn()} onClose={jest.fn()} {...props} />
```